lib/ami_spec/server_spec.rb

```diff
@@ -27,6 +27,13 @@ def run
         puts "Running tests for #{@role}"
       end
 
+      unless Dir.exist?("#{@spec}/#{@role}")
+        puts "WARNING: Role directory #{@spec}/#{@role} does not exist. If you'd like to"
+        puts "skip the role '#{@role}', create the directory but leave it empty"
+        puts "(except for a .gitkeep file)."
+        return false
+      end
```

Comment:
Should this be done during initialization? Doing it during a `run` will spin up other AMIs to test still, despite an imminent failure. Otherwise a `verify` method might be useful?

Reply:
Yeah, I've just moved that in the latest commit.